web/packages/teleport/src/lib/util.ts

```diff
@@ -93,3 +93,39 @@ export function compareByString(a: string, b: string) {
   }
   return 0;
 }
+
+// sortRolesAndFriendlyNames supports roles and their friendly names
+// in the same order.
+export function sortRolesAndFriendlyNames(
```

Comment:
Is there a more appropriate place in the codebase for this? Prior to this change, lib/util.ts was generic sorting and comparison utilities that didn't know anything about Teleport or roles.

Reply:
If this is intended mostly for access requests, `web/packages/teleport/src/AccessRequests/utils.ts` might be a better place for it, as much as I dislike "utils"-style packages/files. ;)

Reply:
After a refactor I don't need this one any more, so I'll remove it.